package.json

```diff
@@ -31,7 +31,8 @@
   ],
   "dependencies": {
     "autoprefixer": "^9.8.4",
-    "broccoli-persistent-filter": "^1.1.6",
+    "broccoli-persistent-filter": "^3.0.0",
+    "chokidar": "^3.0.0",
```

Comment:
It seems odd to add chokidar as a dependency, where previously it was not. Based on your above comment, it would suggest that chokidar is a dependency of our dependencies, which dependency would that be? That answer, will help inform if this should actually be a dependency, devDependency or some other change.